File: 1580-shuffle-the-array/1580-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        lis1=nums[0:n]
        lis2=nums[n:2*n]
        result=[]
        for i in range(n):
            result.append(lis1[i])
            result.append(lis2[i])
        return result

        

File: 1580-shuffle-the-array/1580-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int j=0;
        int k=1;
        int temp = nums[0];
        boolean flag=true;
        int[] result=new int[2*n];
        for(int i=0;i<n;i++)
        {
            result[j]=nums[i];
            result[k]=nums[i+n];
            k+=2;
            j+=2;
        }
        return result;
    }
}